src/Common/src/System/Net/NetworkInformation/UnixCommandLinePing.cs

```diff
@@ -37,6 +41,27 @@ private static string GetPingUtilityPath(bool ipv4)
             return null;
         }
 
+        // Check if found ping is symlink to busybox like alpine /bin/ping -> /bin/busybox
+        private static unsafe bool IsBusyboxPing(string pingBinary)
+        {
+            byte* buffer = stackalloc byte[4000];
```

Comment:
This is a fairly large stack allocation.  While given the use it's unlikely to cause a problem, given that this method is only ever invoked once, I suggest we either just allocate a byte[] or use ArrayPool.